src/main/resources/org/jenkinsci/plugins/pluginusage/PluginUsageView/index.jelly

```diff
@@ -57,6 +58,36 @@
                  </j:forEach>
                  </tbody>
             </table>
+
+            <j:set var="data" value="${it.data}"/>
+            <j:set var="otherPlugins" value="${data.otherPlugins}"/>
+            <br/>
+            <hr/>
+            <h1>${%Other Plugins}</h1>
+            <table class="sortable pane bigtable" border="solid #bbb;">
+                <thead>
+                    <tr>
+                        <th>
+                            ${%Plugin Name}
+                        </th>
+                        <th>
+                            ${%Version}
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <j:forEach var="j" items="${otherPlugins}">
+                        <tr>
+                            <td align="center" style="vertical-align:middle;">
+                                <a href="${j.plugin.url}">${j.longName}</a>
```

Comment:
The link doesn't work, it should be just `${j.url}`

Reply:
nice catch, thanks.  corrected.